### PR TITLE
doc: update CHANGELOG for v1.34.0 release

### PR DESCRIPTION
**Pull request by coryan**
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/7675)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -88,7 +88,57 @@
...
+
+* feat: expose and log ErrorInfo if present  ([#7640](https://github.com/googleapis/google-cloud-cpp/pull/7640))
+* feat(common): add tracing for the LRO polling loop ([#7615](https://github.com/googleapis/google-cloud-cpp/pull/7615))
+* feat(common): add internal-only payload support to Status ([#7603](https://github.com/googleapis/google-cloud-cpp/pull/7603))
+
+### New Libraries
+
+[Cloud Tasks]: https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/tasks/README.md
+[Secret Manager]: https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/secretmanager/README.md
+[Pub/Sub Lite]: https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/secretmanager/README.md
```
Comment (dbolduc) [inline CHANGELOG.md:138]: s/secretmanager/pubsublite/

File: CHANGELOG.md [hunk 2]
```diff
@@ -88,7 +88,57 @@
...
+
+### [Bigtable](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/bigtable/README.md)
+
+* feat(bigtableadmin): add multi-cluster routing for specific clusters ([#7636](https://github.com/googleapis/google-cloud-cpp/pull/7636))
+
+### [Spanner](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/spanner/README.md)
+
+* feat(generator): document details about `*Client` ([#7673](https://github.com/googleapis/google-cloud-cpp/pull/7673))
+* feat(generator): link each method input and output types ([#7665](https://github.com/googleapis/google-cloud-cpp/pull/7665))
+* feat(common): add tracing for the LRO polling loop ([#7615](https://github.com/googleapis/google-cloud-cpp/pull/7615))
```
Comment (devbww) [inline CHANGELOG.md:110]: Already appears in "Common Libraries"?

Comment (coryan) [reply CHANGELOG.md]: Removed, thanks.

**Review by devbww — APPROVED** [review]
For what its worth, LGTM.

**Pull request merged**